webhook.go

```diff
@@ -16,3 +26,45 @@ type Webhook struct {
 	Repo     *library.Repo
 	Build    *library.Build
 }
+
+// ShouldSkip uses the build information
+// associated with the given hook to determine
+// whether the hook should be skipped
+func (w *Webhook) ShouldSkip() (bool, string) {
+	// push event
+	if strings.EqualFold(constants.EventPush, w.Build.GetEvent()) {
+		// the head commit will return null in the hook
+		// payload from the scm when the event is
+		// associated with a branch/tag delete
+		if len(w.Build.GetCommit()) == 0 {
```

Comment:
trying to use what we have available on the hook here. the original payload has some other attributes one could consider, such as the `deleted` field, or you could cue off of the `after` field which will be a blank commit sha consisting of all zeroes (see example payload here: https://developer.github.com/webhooks/event-payloads/#webhook-payload-example-32)